app/frontend/src/resources/logo.svg

```diff
@@ -0,0 +1,10 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 150 150">
+  <g fill="#E47701">
```

Comment:
If I didn't check this on Safari (other browsers happily hide the other redundant bits with `<switch>` for some reason...), didn't even realise how much crap there were in the original logo, including duplicates of it that we don't need!